auto_install/install.sh

```diff
@@ -2225,31 +2393,35 @@ confWireGuard(){
 
 	echo "::: Server Keys have been generated."
 
-	if [ "$pivpnenableipv6" == "1" ]; then
-
-	echo "[Interface]
-PrivateKey = $($SUDO cat /etc/wireguard/keys/server_priv)
-Address = ${vpnGw}/${subnetClass},${vpnGwv6}/${subnetClassv6}
-MTU = ${pivpnMTU}
-ListenPort = ${pivpnPORT}" | $SUDO tee /etc/wireguard/wg0.conf &> /dev/null
+	{
+		echo '[Interface]'
+		echo "PrivateKey = $(${SUDO} cat /etc/wireguard/keys/server_priv)"
+		echo -n "Address = ${vpnGw}/${subnetClass}"
 
-	else
+		if [ "$pivpnenableipv6" == "1" ]; then
+			echo ",${vpnGwv6}/${subnetClassv6}"
+		else
+			echo
+		fi
 
-	echo "[Interface]
-PrivateKey = $($SUDO cat /etc/wireguard/keys/server_priv)
-Address = ${vpnGw}/${subnetClass}
-MTU = ${pivpnMTU}
-ListenPort = ${pivpnPORT}" | $SUDO tee /etc/wireguard/wg0.conf &> /dev/null
+		echo "MTU = ${pivpnMTU}"
+		echo "ListenPort = ${pivpnPORT}"
+	} | ${SUDO} tee /etc/wireguard/wg0.conf &> /dev/null
 
-	fi
 	echo "::: Server config generated."
 }
 
 confNetwork(){
 	# Enable forwarding of internet traffic
-	$SUDO sed -i '/net.ipv4.ip_forward=1/s/^#//g' /etc/sysctl.conf
+	if ! $SUDO sed -i -E -e 's/^#(net\.ipv4\.ip_forward)=.*/\1=1/g' /etc/sysctl.conf; then
+		echo 'net.ipv4.ip_forward=1' | $SUDO tee -a /etc/sysctl.conf > /dev/null
+	fi
+
 	if [ "$pivpnenableipv6" == "1" ]; then
-		$SUDO sed -i '/net.ipv6.conf.all.forwarding=1/s/^#//g' /etc/sysctl.conf
+		if ! $SUDO sed -i -E -e 's/^#(net\.ipv6\.conf\.all\.forwarding)=.*/\1=1/g' /etc/sysctl.conf; then
+			echo 'net.ipv6.conf.all.forwarding=1' | $SUDO tee -a /etc/sysctl.conf > /dev/null
+		fi
+
 		echo "net.ipv6.conf.${IPv6dev}.accept_ra=2" | $SUDO tee /etc/sysctl.d/99-pivpn.conf > /dev/null
 	fi
 	$SUDO sysctl -p > /dev/null
```

Comment:
Let's make it easy instead of fancy (still didn't work because sed wouldn't return an error).

```suggestion
	# Enable forwarding of internet traffic
	echo 'net.ipv4.ip_forward=1' | $SUDO tee /etc/sysctl.d/99-pivpn.conf > /dev/null

	if [ "$pivpnenableipv6" == "1" ]; then
		echo "net.ipv6.conf.all.forwarding=1
net.ipv6.conf.eth0.accept_ra=2" | $SUDO tee -a /etc/sysctl.d/99-pivpn.conf > /dev/null
	fi

	${SUDO} sysctl -p /etc/sysctl.d/99-pivpn.conf > /dev/null
```